js/src/setup-mc/setup-stepper/setup-paid-ads/skip-paid-ads-confirmation-modal.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { select } from '@wordpress/data';
+import { createInterpolateElement } from '@wordpress/element';
+import { noop, merge } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import AppModal from '.~/components/app-modal';
+import AppButton from '.~/components/app-button';
+import AppDocumentationLink from '.~/components/app-documentation-link';
+import useGoogleAdsAccount from '.~/hooks/useGoogleAdsAccount';
+import { ACTION_SKIP } from './constants';
+import { STORE_KEY } from '.~/data/constants';
+
+/**
+ * Triggered when the skip button is clicked
+ * // TODO: to review
+ *
+ * @event gla_onboarding_complete_button_click
+ */
+
+/**
+ * Renders a modal dialog that confirms whether the user wants to skip setting up paid ads.
+ * It provides information about the benefits of enabling Performance Max and includes a link to learn more.
+ *
+ * @param {Object} props React props.
+ * @param {Function} props.onRequestClose Function to be called when the modal should be closed. Defaults to a no-op function.
+ * @param {Function} props.onSkipConfirmation Function to be called when the user confirms skipping the paid ads setup. Defaults to a no-op function.
+ * @param {boolean} [props.isProcessing=false] Indicates whether a process is currently running (e.g., the confirmation is being processed). If true, the confirmation button will show a loading state.
+ * @param {boolean} [props.showPaidAdsSetup=false] Indicates whether the paid ads setup is currently shown. If true, additional event properties will be included in the eventProps.
+ * @param {Object} [props.paidAds={}] The paid ads data, including the campaign form data and validation status.
+ */
+const SkipPaidAdsConfirmationModal = ( {
+	onRequestClose = noop,
+	onSkipConfirmation = noop,
+	isProcessing = false,
+	showPaidAdsSetup = false,
+	paidAds = {},
+} ) => {
+	const { googleAdsAccount } = useGoogleAdsAccount();
+
+	const eventProps = {
+		opened_paid_ads_setup: 'no',
+		google_ads_account_status: googleAdsAccount?.status,
+		billing_method_status: 'unknown',
+		campaign_form_validation: 'unknown',
+	};
+
+	// TODO: Review once https://github.com/woocommerce/google-listings-and-ads/issues/2500 is merged
+	if ( showPaidAdsSetup ) {
+		const selector = select( STORE_KEY );
+		const billing = selector.getGoogleAdsAccountBillingStatus();
+
+		merge( eventProps, {
+			opened_paid_ads_setup: 'yes',
+			billing_method_status: billing?.status,
+			campaign_form_validation: paidAds.isValid ? 'valid' : 'invalid',
+		} );
+	}
+
+	return (
+		<AppModal
+			className="gla-ads-skip-paid-ads-modal"
```

Comment:
The `gla-ads-skip-paid-ads-modal` class name seems unnecessary?

Reply:
Removed @eason9487